webpack.commons.js

```diff
@@ -16,7 +16,7 @@ const pugLoaderOptions = {
   root: `${__dirname}/node_modules/esn-frontend-common-libs/src/frontend/views`
 };
 
-const BASE_HREF = process.env.BASE_HREF || '/';
+const BASE_HREF = process.env.BASE_HREF || '/contacts';
```

Comment:
The default should end with a `/`
```suggestion
const BASE_HREF = process.env.BASE_HREF || '/contacts/';
```